atd-vzd/schema/council_districts.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE public.council_districts(
+    id SERIAL PRIMARY KEY,
```

Comment:
shoutout @johnclary for the sly [tip](https://github.com/cityofaustin/atd-vz-data/pull/1234#discussion_r1219806164) last time